node-js-basics-101/node_modules/negotiator/index.js

```diff
@@ -0,0 +1,124 @@
+/*!
+ * negotiator
+ * Copyright(c) 2012 Federico Romero
+ * Copyright(c) 2012-2014 Isaac Z. Schlueter
+ * Copyright(c) 2015 Douglas Christopher Wilson
+ * MIT Licensed
+ */
+
+'use strict';
```

Comment:
__[jshint]__
* Use the function form of "use strict".

<!-- Comment by [SideCI](https://sideci.com) -->